x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/test_pipeline.tsx

```diff
@@ -7,6 +7,112 @@
+    simulatePipelineErrors,
+  } = useValues(MLInferenceLogic);
+  const { simulatePipeline, setPipelineSimulateBody } = useActions(MLInferenceLogic);
+
+  const isSmallerViewport = useIsWithinMaxBreakpoint('s');
+
+  return (
+    <EuiFlexGroup direction="column" gutterSize="xs">
+      <EuiFlexItem>
+        <EuiText>
+          <h4>
+            {i18n.translate(
+              'xpack.enterpriseSearch.content.indices.pipelines.addInferencePipelineModal.steps.test.title',
+              { defaultMessage: 'Review pipeline results (optional)' }
+            )}
+          </h4>
+        </EuiText>
+      </EuiFlexItem>
+      <EuiFlexItem>
+        <EuiResizableContainer
+          direction={isSmallerViewport ? 'vertical' : 'horizontal'}
+          className="resizableContainer"
+        >
+          {(EuiResizablePanel, EuiResizableButton) => (
+            <>
+              <EuiResizablePanel grow hasBorder initialSize={50} paddingSize="xs">
+                <CodeEditor
+                  languageId="json"
+                  options={{
+                    automaticLayout: true,
+                    lineNumbers: 'off',
+                    tabSize: 2,
+                  }}
+                  value={simulateBody}
+                  onChange={(value) => {
+                    setPipelineSimulateBody(value);
+                  }}
+                />
+              </EuiResizablePanel>
+
+              <EuiResizableButton />
+
+              <EuiResizablePanel grow hasBorder initialSize={50} paddingSize="xs">
+                <EuiCodeBlock language="json" isCopyable className="reviewCodeBlock">
+                  {simulatePipelineErrors.length > 0
+                    ? JSON.stringify(simulatePipelineErrors, null, 2)
+                    : JSON.stringify(simulatePipelineResult || '', null, 2)}
+                </EuiCodeBlock>
+              </EuiResizablePanel>
+            </>
+          )}
+        </EuiResizableContainer>
+      </EuiFlexItem>
+      <EuiSpacer />
+      <EuiFlexItem grow={false}>
+        <EuiFlexGroup alignItems="center" justifyContent="flexEnd" gutterSize="xs">
+          <EuiFlexItem>
+            <EuiText>
+              <p>
+                {i18n.translate(
+                  'xpack.enterpriseSearch.content.indices.pipelines.addInferencePipelineModal.steps.test.description',
+                  {
+                    defaultMessage:
+                      'You can simulate your pipeline results by passing an array of documents.',
+                  }
+                )}
+              </p>
+            </EuiText>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <div>
+              <EuiButton onClick={simulatePipeline}>
+                {i18n.translate(
+                  'xpack.enterpriseSearch.content.indices.pipelines.addInferencePipelineModal.steps.test.runButton',
+                  { defaultMessage: 'Simulate Pipeline' }
+                )}
+              </EuiButton>
```

Comment:
Because this button's been added, we'll need to change the "Continue" button in the footer to a filled button so that it's the solid bright blue color. That way we don't have buttons competing for attention. 🙂 
<img width="866" alt="image" src="https://user-images.githubusercontent.com/739960/195411994-cd6b6794-8566-4a5e-8bca-f6db55e029ad.png">

Reply:
Great, I made that button filled, AND the create button as well to stay consistent. 

<img src="https://user-images.githubusercontent.com/1410658/195572424-36fb095b-5320-4fe6-aa44-e78a8823d31a.png"/>
<img src="https://user-images.githubusercontent.com/1410658/195572427-0e90470b-7550-42c9-9d20-876a01907e9a.png" />
<img src="https://user-images.githubusercontent.com/1410658/195572429-10e5fe60-a1eb-4328-8cf3-a06e978c59e2.png" />


Otherwise the last step looks like this
![Screenshot 2022-10-13 at 12 22 38](https://user-images.githubusercontent.com/1410658/195573298-11b3769d-c69e-47b0-8093-70d5d59865a4.png)